Some missing traits for Radial intervals at 0 (S135) (#1191)
pi-base/data: spaces/S000135/properties/P000080.md

---
space: S000135
property: P000080
value: true
refs:
- doi: 10.1007/978-1-4612-6290-9_6
  name: Counterexamples in Topology
---

> Take $A\subset X$ and $p\in \overline A$. If $p\neq(0,0)$ then it admits countable base of neighbourhoods and picking elements of $A$ from each of those neighbourhoods we obtain the required sequence.
> For $p=(0,0)$, observe that there has to exists at least one ray
> containg an element of $A$ arbitrarily close to $p$. Otherwise a neighbourhood of the origin dijsoint with $A$ can be found.

Reviewer: ```suggestion
Take $A\subseteq X$ and $p\in \overline A\setminus A$. If $p\neq(0,0)$ then it admits a countable base of neighbourhoods and picking elements of $A$ from each of those neighbourhoods we obtain the required sequence.
For $p=(0,0)$, there has to exist at least one ray
containing elements of $A$ arbitrarily close to $p$. Otherwise a neighbourhood of the origin disjoint from $A$ could be found.
```
